### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by aimalamiri**
# Designed the desktop version of the entire page
### All of the bellow sections are designed for desktop version and the web page is now completely responsive
1. Designed Navbar Section
2. Designed Header Section
3. Designed Works Section
4. Designed About Section
5. Designed Contact Section

And I'm done 😊

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,8 +31,8 @@
       </ul>
       <a href="#" class="navbar-menu-button"><img src="./assets/svg/menu.svg" alt="Menu" /></a>
     </nav>
-    <header class="header container">
-      <section class="header-content">
+    <header class="header">
+      <section class="header-content container">
         <h1 class="header-title font-crete-round text-orange">Hey There. I'm Aimal</h1>
```
Comment (Kingstalux) [inline index.html:38]: - [x] Good job so far 🔥 your design looks great and it follows the template. Kindly align the social links to the right in the desktop version as shown in the template 👍 
- Your design:
Annotation 2022-03-23 195948
- Figma design:
Annotation 2022-03-23 200000

File: styles.css [hunk 2]
```diff
@@ -400,6 +400,88 @@ header {
...
+    width: calc(100% - 648px);
+  }
+
+  .f-project-body > .project-title {
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 0;
+    color: #091e42;
+  }
+
```
Comment (Kingstalux) [inline styles.css:463]: - Good job aligning your project cards 👍 There are a few issues to address here. 
- [x] Each project should have their description on top of their image with a dark opacity and with a white color text as shown in the second project card. 
- [x] When the project is hovered on with the mouse the description and dark opacity disappear and a `See Project` button appears to the bottom of the image. Kindly implement this feature so as to follow the template design. Hence all projects should look like the first project card when you hover over it.
- Your design:
Annotation 2022-03-23 200025
- Figma design:
Annotation 2022-03-23 200039

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @aimalamiri and @Enekwestanley, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966


## Highlights
✅ Good pull request description
✅ Great design 🔥 
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @aimalamiri, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**